asf_heat_pump_suitability/pipeline/flows/run_calculate_suitability_flow.py

```diff
@@ -0,0 +1,283 @@
+"""
+Calculate suitability scores of different low-carbon heating technologies in Nesta and 'conventional' views for individual
+properties and LSOAs.
+
+To run:
+python asf_heat_pump_suitability/pipeline/flows/run_calculate_suitability_flow.py --datastore=s3 run --weights [path/to/weighted/EPC] --features [path/to/EPC/with/features] --gardens [path/to/garden/size/estimates] --year [YYYY] --quarter [Q] --max-workers 18
```

Comment:
Update doc string to include sample parameter
```suggestion
python asf_heat_pump_suitability/pipeline/flows/run_calculate_suitability_flow.py --datastore=s3 run --weights [path/to/weighted/EPC] --features [path/to/EPC/with/features] --gardens [path/to/garden/size/estimates] --year [YYYY] --quarter [Q] --max-workers 18 --parameter True
```